src/config.ts

```diff
@@ -196,6 +264,16 @@ export const Config = Schema.intersect([
       endpoint: Schema.string().description('API 服务器地址。').required(),
       headers: Schema.dict(String).description('要附加的额外请求头。'),
     }),
+    Schema.object({
+      type: Schema.const('stable-horde'),
+      endpoint: Schema.string().description('API 服务器地址。').default('https://stablehorde.net/'),
+      token: Schema.string().description('授权令牌。').role('secret').default('0000000000'),
+      nsfw: Schema.union([
+        Schema.const('disallow').description('禁止'),
+        Schema.const('censor').description('遮罩'),
+        Schema.const('allow').description('允许'),
+      ]).description('是否允许 NSFW 内容。').default('allow'),
```

Comment:
- Default to `allow` may be arguable.
- Don't know what is the translation of `censor` (or `censored`?)